x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1217,3 +1217,12 @@ FROM airports
 c:l
 891
 ;
+
+countMV#[skip:-8.13.99,reason:supported in 8.14]
+FROM employees
+| STATS vals = COUNT(salary_change.int)
+;
+
+vals:l
+183
```

Comment:
Without this fix, this becomes 79 - the number of docs with non-null `salary_change`; but only in `EsqlSpecIT`, in the CSV tests (which don't push down to Lucene) this is always 183!